crawl-ref/source/dat/des/arrival/small.des

```diff
@@ -3412,3 +3412,126 @@ MAP
   ...........
    .........
 ENDMAP
+
+##############################################################
+NAME:    heliophobic_arrival_battle_scene
+TAGS:    arrival no_monster_gen no_trap_gen no_pool_fixup
+ORIENT:  float
+ITEM:    human skeleton / dwarf skeleton , \
+         goblin skeleton / gnoll skeleton / \
+         kobold skeleton / orc skeleton
+SHUFFLE: cx
+SHUFFLE: tvw
+FTILE:   .{det+ = floor_marble
+MAP
+cccc+cccc
+ce..de.ec
+c.e.w..ec
+cd..we..c
+c.d.w.d.c
+c..e..e.c
+ced.{d.ec
+c.d..de.c
+ccccccccc
+ENDMAP
+
+##############################################################
+NAME:    heliophobic_arrival_enchanted_forest
+TAGS:    arrival no_monster_gen no_trap_gen
+ORIENT:  float
+SHUFFLE: TAG
+FTILE:  .{TAGtF = floor_moss
+MARKER:  F = lua:fog_machine{ \
+             pow_max = 20, delay_min = 20, delay_max = 200, \
+             size = 5, start_clouds = 4, \
+             cloud_type = "black smoke"}
+MAP
+     .tt.t
+  Tt.t.t.tt.F
+ F.tt.A.At.t.t
+ tt.tA...A.tt.
+..t.A..G..Att.t
+tt.A.......A.tt
+.tt..G.{.G....@
+t.tA.......At.t
+t.t.A..G..A.t.t
+ tt.tA...At.tt
+ ..tt.A.At.t.t
+  .F.tt..t.TF
+     t.tt.
+ENDMAP
+
+##############################################################
+NAME:    heliophobic_arrival_ceremonial_halls
+TAGS:    arrival no_monster_gen no_trap_gen
+ORIENT:  float
+FTILE:  .+{>G = floor_tomb
+MAP
+>....vvv+vvv....>
+.....vG...Gv.....
+..v..vv.v.vv..v..
+.vvv.vG.v.Gv.vvv.
+.vvv.vv.v.vv.vvv.
+..v..vG.v.Gv..v..
+.....vv.v.vv.....
+vvvvvvG...Gvvvvvv
+vGvGvGv...vGvGvGv
+v...............v
++.vvv...{...vvv.+
+v...............v
+vGvGvGv...vGvGvGv
+vvvvvvG...Gvvvvvv
+.....vv.v.vv.....
+..v..vG.v.Gv..v..
+.vvv.vv.v.vv.vvv.
+.vvv.vG.v.Gv.vvv.
+..v..vv.v.vv..v..
+.....vG...Gv.....
+>....vvv+vvv....>
+ENDMAP
+
+##############################################################
+NAME:    heliophobic_arrival_henge
+TAGS:    arrival no_monster_gen no_trap_gen
+ORIENT:  float
+FTILE:  .t{A = floor_moss
+MAP
+    ttttt
+  ttttttttt
+ ttttAAAtttt
+ ttA.....Att
+ttA..A.A..Att
+ttA...{.....@
+ttA..A.A..Att
+ ttA.....Att
+ ttttAAAtttt
+  ttttttttt
+    ttttt
+ENDMAP
+
+##############################################################
+NAME:    heliophobic_arrival_glass_houses
+TAGS:    arrival no_monster_gen no_pool_fixup no_trap_gen
+ORIENT:  float
+KPROP:   w = no_tele_into
```

Comment:
Players won't teleport into something that will kill them; deep water does not need no_tele_into

Reply:
Making deep water `no_tele_into` is not how we use the `no_tele_into` flag; it is used to keep players out of teleport closets, which this deep water is not.